Neos.Flow/Classes/Mvc/Routing/RouterCachingService.php

```diff
@@ -206,6 +206,17 @@ public function flushCachesByTag($tag)
         $this->resolveCache->flushByTag($tag);
     }
 
+    /**
+     * Flushes 'findMatchResults' and 'resolve' caches for the given $tags
+     *
+     * @param array<string> $tags
+     */
+    public function flushCachesByTags(array $tags): void
```

Comment:
if we introduce this method, we should IMO use it in the core and deprecate (or even remove) `flushCachesByTag()` but I would suggest to do this in a separate PR

Reply:
It is only used in Neos.Neos and contained in the PR https://github.com/neos/neos-development-collection/pull/3631

I don't see the necessity to remove the option to flush a single tag.